lighthouse-core/gather/driver/prepare.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const log = require('lighthouse-logger');
+const storage = require('./storage.js');
+const emulation = require('../../lib/emulation.js');
+const pageFunctions = require('../../lib/page-functions.js');
+
+/**
+ * Enables `Debugger` domain to receive async stacktrace information on network request initiators.
+ * This is critical for tracking attribution of tasks and performance simulation accuracy.
+ * @param {LH.Gatherer.FRProtocolSession} session
+ */
+async function enableAsyncStacks(session) {
+  const enable = async () => {
+    await session.sendCommand('Debugger.enable');
+    await session.sendCommand('Debugger.setSkipAllPauses', {skip: true});
+    await session.sendCommand('Debugger.setAsyncCallStackDepth', {maxDepth: 8});
+  };
+
+  // Resume any pauses that make it through `setSkipAllPauses`
+  session.on('Debugger.paused', () => session.sendCommand('Debugger.resume'));
+
+  // `Debugger.setSkipAllPauses` is reset after every navigation, so retrigger it on main frame navigations.
+  // See https://bugs.chromium.org/p/chromium/issues/detail?id=990945&q=setSkipAllPauses&can=2
+  session.on('Page.frameNavigated', event => {
```

Comment:
this little 4 lines is new based on https://bugs.chromium.org/p/chromium/issues/detail?id=990945&q=setSkipAllPauses&can=2